core/nwb.ophys.yaml

```diff
@@ -1,4 +1,50 @@
 groups:
+- neurodata_type_def: OnePhotonSeries
+  neurodata_type_inc: ImageSeries
+  doc: Image stack recorded over time from 1-photon microscope.
+  attributes:
+  - name: pmt_gain
+    dtype: float32
+    doc: Photomultiplier gain.
+    required: false
+  - name: scan_line_rate
+    dtype: float32
+    doc: Lines imaged per second. This is also stored in /general/optophysiology but
+      is kept here as it is useful information for analysis, and so good to be stored
+      w/ the actual data.
+    required: false
+  - name: exposure_time
+    dtype: float32
+    doc: Exposure time of the sample; often the inverse of the frequency.
+    required: false
+  - name: binning
+    dtype: uint8
+    doc: Amount of pixels combined into 'bins'; could be 1, 2, 4, 8, etc.
+    required: false
```

Comment:
Could you help me understand what the purpose of the ``binning` attribute is and how the value is being used? I think part of my confusion is that I'm not sure what ``combined into 'bins'`` is referring to. Is the term ``bins`` here used to refer to a pixel in the actual data, where the imaging device acquires multiple pixels and then stores some weighted average for the pixel? If that is case, is knowing the number of pixels that were combined sufficient or do you need to know the matrix with the weights?

Reply:
Thanks for the clarification